### PR TITLE
feat(helm): add liveness probe to pdfgenerator

### PR DESCRIPTION
**Pull request by tcrasset**
Closes https://github.com/octopize/avatar/issues/1351

### CODE REVIEW

File: bin/minikube/Makefile [hunk 1]
```diff
@@ -27,7 +43,7 @@ lint-fix:  ## Run autoformatters
 .PHONY: lint-fix
 
 typecheck:  ## Run typechecking
-	poetry run mypy --show-error-codes --pretty --show-traceback .
+	poetry run mypy --show-error-codes --pretty --show-traceback . --config ../pyproject.toml
```
Comment (tcrasset) [inline bin/minikube/Makefile:46]: Somehow `mypy` was not finding the correct config, I think it was looking at `setup.cfg`

File: bin/minikube/launch.py [hunk 2]
```diff
@@ -1078,31 +1082,44 @@ def verify_api_health(
...
+    auth_password: Optional[str] = None
+    if use_email_authentication:
+        email_or_username = cast(
+            EmailAuthentication, config.authentication
+        ).admin_emails[0]
+    else:
+        auth = cast(UsernameAuthentication, config.authentication)
+        email_or_username = auth.username
+        auth_password = auth.password
+
```
Comment (tcrasset) [inline bin/minikube/launch.py:1110]: Typechecking

Comment (tcrasset) [reply bin/minikube/launch.py:1110]: The rest is linting

**Review by tas17 — APPROVED** [review]
LGTM :)

**Pull request merged**